README.md

```diff
@@ -0,0 +1,26 @@
+## Kasse
+
+This is an Implementation of a payment system for the
+[Heidelberg Chaostreff](https://www.noname-ev.de). It is currently in a basic
+stage of devellopment, so this README.md should give you enough information to
+install it and start testing.
+
+## Installing
+
+Currently you need to install libnfc to build kasse. This will not be needed in
+the future. In Debian you need to install the following packages:
+
+- libnfc-dev
+- sqlite3
+- golang
+
+
```

Comment:
Some comment seems in order here. Like "The following will build the code and give you a basic environment to run kasse".